lang/csharp/CodeMaid.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need this file? It seems user specific

Reply:
File was deleted